specification/web/resource-manager/Microsoft.DomainRegistration/stable/2024-11-01/TopLevelDomains.json

```diff
@@ -0,0 +1,303 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-11-01",
+    "title": "TopLevelDomains API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DomainRegistration/topLevelDomains": {
+      "get": {
+        "tags": [
+          "TopLevelDomains"
+        ],
+        "summary": "Get all top-level domains supported for registration.",
+        "description": "Description for Get all top-level domains supported for registration.",
+        "operationId": "TopLevelDomains_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/TopLevelDomainCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Top Level Domains": {
+            "$ref": "./examples/ListTopLevelDomains.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DomainRegistration/topLevelDomains/{name}": {
+      "get": {
+        "tags": [
+          "TopLevelDomains"
+        ],
+        "summary": "Get details of a top-level domain.",
+        "description": "Description for Get details of a top-level domain.",
+        "operationId": "TopLevelDomains_Get",
+        "parameters": [
+          {
+            "name": "name",
```

Comment:
>name [](http://example.com/codeflow?start=21&length=4)

Use a more specific name, e.g. domainName. Feedback applies throughout this file.